.buildkite/pipeline.yml

```diff
@@ -102,74 +98,80 @@ steps:
     steps:
       - label: "Global Bucket on CPU (functional albedo)"
         key: "global_bucket_function_cpu"
-        command: "julia --color=yes --project=experiments experiments/standalone/Bucket/global_bucket_function.jl"
+        command: "julia --color=yes --project=.buildkite experiments/standalone/Bucket/global_bucket_function.jl"
         artifact_paths: "experiments/standalone/Bucket/artifacts/*cpu*"
 
       - label: "Global Bucket on CPU (static map albedo)"
         key: "global_bucket_staticmap_cpu"
-        command: "julia --color=yes --project=experiments experiments/standalone/Bucket/global_bucket_staticmap.jl"
+        command: "julia --color=yes --project=.buildkite experiments/standalone/Bucket/global_bucket_staticmap.jl"
         artifact_paths: "experiments/standalone/Bucket/artifacts_staticmap/*cpu*"
 
       - label: "Global Bucket on CPU (temporal map albedo)"
         key: "global_bucket_temporalmap_cpu"
-        command: "julia --color=yes --project=experiments experiments/standalone/Bucket/global_bucket_temporalmap.jl"
+        command: "julia --color=yes --project=.buildkite experiments/standalone/Bucket/global_bucket_temporalmap.jl"
         artifact_paths: "experiments/standalone/Bucket/artifacts_temporalmap/*cpu*"
 
   - group: "GPU: unit tests and global bucket"
     steps:
       - label: "GPU runtests"
-        command: "julia --color=yes --project=test test/runtests.jl"
+        command: "julia --color=yes --project=.buildkite test/runtests.jl"
         agents:
           slurm_ntasks: 1
           slurm_gres: "gpu:1"
 
       - label: "soil/canopy lsm performance on GPU"
-        command: "julia --color=yes --project=experiments experiments/integrated/performance/profile_allocations.jl"
+        command: "julia --color=yes --project=.buildkite experiments/integrated/performance/profile_allocations.jl"
         agents:
           slurm_ntasks: 1
           slurm_gres: "gpu:1"
         artifact_paths: "experiments/integrated/performance/*gpu*html"
 
       - label: "Global Bucket on GPU (functional albedo)"
         key: "global_bucket_function_gpu"
-        command: "julia --color=yes --project=experiments experiments/standalone/Bucket/global_bucket_function.jl"
+        command: "julia --color=yes --project=.buildkite experiments/standalone/Bucket/global_bucket_function.jl"
         agents:
           slurm_ntasks: 1
           slurm_gres: "gpu:1"
+        env:
+          CLIMACOMMS_DEVICE: "CUDA"
```

Comment:
Is this needed for the ClimaComms/ClimaCore update? Just wondering because I'll probably have to do the same in ClimaCoupler

Reply:
Yes, it is needed. More information here: https://github.com/CliMA/ClimaComms.jl/wiki/Transition-to-ClimaComms.jl-0.6